### PR TITLE
LPS-195201 Use setURLBackTitle parameter to indicate the title in the knowledge-base-web module 

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The motivation behind this ticket is to use setURLBackTitle method for setting the name of the page when hovering over the back button to give the user more context.
https://liferay.atlassian.net/browse/LPS-195201

Proposed Solution
========
What I did to solve this problem is to use the new setURLBackTitle in the PortletDisplay class for setting this parameter.

Steps to Verify
========
1. Go to Knowledge Base -> Create a KB and go to the View and Edit views and. check the back title.
2. Repeat the same with folders, sugestions and templates.
Screenshot 2023-09-05 at 15 37 48

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp [hunk 1]
```diff
@@ -10,9 +10,12 @@
...
 EditKBArticleDisplayContext editKBArticleDisplayContext = new EditKBArticleDisplayContext(kbGroupServiceConfiguration, liferayPortletRequest, liferayPortletResponse, portletConfig);
 
+KBArticle kbArticle = editKBArticleDisplayContext.getKBArticle();
+
 if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 	portletDisplay.setShowBackIcon(true);
 	portletDisplay.setURLBack(editKBArticleDisplayContext.getRedirect());
+	portletDisplay.setURLBackTitle(portletDisplay.getTitle());
 
 	renderResponse.setTitle(editKBArticleDisplayContext.getHeaderTitle());
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp:20]: Isn't the `kbArticle` variable unused? Why are you creating it?

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by austinchiang** [comment]
Test failures unrelated

**Comment by austinchiang** [comment]
ci:forward:force

**Pull request closed without merging**